examples/postgres/aurora/README.md

```diff
@@ -11,6 +10,30 @@ resources:
     type: postgres
 ```
 
+## Infrastructure setup
+
+```mermaid
+graph TD;
+  subgraph VPC
+    database["Postgres AWS RDS Aurora instance"]
+    sg["Postgres security group"]
```

Comment:
Going through the security group looks a bit strange, maybe make the security group an annotation on the arrow?

Reply:
Hmm, you're right. Annotation looks better!